dialect/pgdialect/range.go

```diff
@@ -1,138 +1,156 @@
 package pgdialect
 
 import (
+	"bytes"
 	"database/sql"
+	"database/sql/driver"
 	"fmt"
 	"io"
-	"time"
+	"reflect"
 
-	"github.com/uptrace/bun/internal"
 	"github.com/uptrace/bun/schema"
 )
 
 type MultiRange[T any] []Range[T]
 
+var (
+	_ sql.Scanner   = (*MultiRange[any])(nil)
+	_ driver.Valuer = (*MultiRange[any])(nil)
+)
+
+func (m *MultiRange[T]) Scan(anySrc any) (err error) {
+	return Array(m).Scan(anySrc)
+}
+
+func (m MultiRange[T]) Value() (driver.Value, error) {
+	return m.String(), nil
+}
+
+func (m MultiRange[T]) String() string {
+	if len(m) == 0 {
+		return "{}"
+	}
+	var b []byte
+	b = append(b, '{')
+	for _, r := range m {
+		b = append(b, unquote(appendElem(nil, r))...)
```

Comment:
unquote is not the most elegant and performant solution. And it seems to be in other places as well.